### PR TITLE
Parallel attestors per type

### PR DESCRIPTION
**Pull request by matglas**
Allowing per attestor type to run all the attestors in parallel to speed up the process.

- [x] Check for need of mutex in different attestors when using data from context.

### CODE REVIEW

**Comment by ChaosInTheCRD** [comment]
I seem to recall that there was a reason why we opted to not run attestors in parallel 🤔 @mikhailswift @jkjell any ideas?

File: attestation/context.go
Comment (matglas) [inline attestation/context.go:141]: Thinking of adjusting the log line here to make it clear that this is the start of a phase. And maybe add a Name method to OrderType so we can give a better visual feedback.

File: attestation/context.go
Comment (matglas) [inline attestation/context.go:187]: Thinking of making this a `Infof` instead of `Debugf` so you get more understandable output.

**Review by jkjell — APPROVED** [review]
Given the scope of the changes, I'll wait for a second approval from @ChaosInTheCRD or @mikhailswift before merging.

**Comment by matglas** [comment]
Nice thanks!

**Pull request merged**
